sections/Abstract.tex

```diff
@@ -0,0 +1,14 @@
+Being able to perform accurate time series forecasting has always played a big role for humanity, especially when it comes to predicting the weather. 
+Today, the state of the art methods for achieving the best possible results employ machine learning. 
+For many years, however, machine and deep learning techniques have had trouble dealing with large datasets. 
+More recently, the transformer model has shown great potential handling large datasets in the field of natural language processing.
+
+In this paper, we aim to modify the already developed transformer model such that it may be used for time series forecasting. 
+Because we are not working with natural language processing, we propose the removal of the decoder layer, since we do not to output any decoded data. 
```

Comment:
it's not really our invention, so not sure we can claim that it's our proposal. our invention is the interaction with the system (web-app)